liblouis/compileTranslationTable.c

```diff
@@ -4808,6 +4808,11 @@ freeTranslationTable(TranslationTableHeader *t) {
 	free(t);
 }
 
+static void
```

Comment:
Seems a bit excessive to create a separate function for this, but fine with me.
You could move the `if not(null)` test in here

Reply:
I just mimicked `freeTranslationTable` without much thinking actually. The idea was that `freeDisplayTable` could in theory do more freeing, like `freeTranslationTable`. The caller doesn't need to know.

Regarding the null check: `freeTranslationTable` also doesn't do it. In cases like these, with a single argument, I think it makes most sense to assume the caller knows the table is not null, otherwise it shouldn't call it. I wouldn't move the `if` inside the function just to make it more meaningful. In a language like Java I might do a check and raise an exception to catch coding errors.